changelogs/current.yaml

```diff
@@ -83,3 +80,6 @@ new_features:
     added support for lmove command.
 
 deprecated:
+- area: tracing
+  change: |
+    OpenTracing is deprecated and will be removed at version 2.10, since the upstream project has been abandoned.
```

Comment:
How 'bout we slightly rephrase this to be:
```
OpenTracing is deprecated and will be removed, since the upstream project has been abandoned. OpenTracing 2.10 is the last version which will be imported into Envoy.
```

Reply:
i think there is not much point putting a deprecation notice if it doesnt say _when_ the thing will be removed - its pretty unhelpful

i have put version 2.10 on the basis that this is the first version after the 6 month cutoff

Reply:
envoy version

Reply:
as discussed offline - im getting totally confused here - the removal version should be `1.30` not `2.10`

ill update